composer.json

```diff
@@ -60,6 +60,8 @@
         "drupal/mysql56": "^1.0",
         "drupal/next": "1.0.0",
```

Comment:
@modularlab Sorry, I missed this in earlier review. Can we unpin this module version ?

>If any patches needs to be added, then pin the module version (instead of using min. stability version. Ex: ‘drupal/next:^1.0’ shouldn’t be used instead ‘drupal/next:1.0.0’ should be used). 

By this, what I mean is: If we are adding patch for the module which we've added in composer.json, then we should pin the module version, else it's not needed. We haven't any patches for `drupal/next` module, we've added patch for modules: `drupal/decoupled_router` & `drupal/subrequests`. So, we don't need to pin `drupal/next` module.

Reply:
No worries.  I've actually addressed this in #1098.  I'll also hold off on fixing any composer conflicts on this PR until #1098 has been merged.